README.md

```diff
@@ -11,6 +11,8 @@ Firmware for Muscle BioAmp hardware from Upside Down Labs
 |6 | [LEDBarGraph](6_LEDBarGraph)| LED bar graph representing EMG Amplitude.|
 |7 | [MuscleStrengthGame](7_MuscleStrengthGame)| Measure your muscle strength. |
 |8 | [EMGScrolling](8_EMGScrolling)| Scrolling through screen using hands. |
+|9 | [2Channel_LCDBarGraph](9_2Channel_LCDBarGraph)| LCD bar graph representing EMG Amplitude using 2 Channel |
```

Comment:
We can create a diagram to show how Muscle BioAmp Shield and Muscle BioAmp Pathy were connected to use this program.

Reply:
I have added a jpeg image file showing the 2 Channel EMG Signal Acquisition to describe the appropriate connections with pins and ports